graphql/pupil/fields.ts

```diff
@@ -110,4 +110,18 @@ export class ExtendFieldsPupilResolver {
     async canJoinSubcourses(@Root() pupil: Required<Pupil>) {
         return canJoinSubcourses(pupil);
     }
-}
\ No newline at end of file
+
+    @FieldResolver(returns => [Boolean])
+    @Authorized(Role.ADMIN)
+    async isParticipant(@Root() pupil: Required<Pupil>, @Arg("subcourseId") subcourseId: number) {
+        return await prisma.subcourse_participants_pupil.count({ where: { pupilId: pupil.id, subcourseId }}) > 0 ? true : false;
```

Comment:
? true : false is useless ;)

Reply:
you sure? I think i tested it and it didnt work when returning the count. I think something with "iterable" was the error. But I think I will try again

Reply:
Yeah but the result of the comparison with zero (>0) is already a boolean :)

Reply:
lol I am stupid hahaha